### DISCUSSION TITLE
How to setup Shoutrrr?

### DISCUSSION THREAD

**Discussion by Savasdotexe** [qdm12/ddns-updater#686] [Q&A]
I've installed ddns-updater via TrueNAS Scale Apps, (App Version: 2.6.0, Chart Version: 1.0.27). I've setup two domains to update, however I can't seem to figure out how to setup/enable Shoutrrr to send notifications.

I'd like to send a simple notification of when an IP has been updated or failed to update. After looking at the [Shoutrrr services overview](https://containrrr.dev/shoutrrr/v0.8/services/overview/), for now email my preferred. But I'm not sure what to do on ddns-updater to get it done..

I've been looking at https://containrrr.dev/shoutrrr/v0.8/services/email/ however that's the only documentation/instructions I can find, so I'm basically unsure as I've never set something similar up before.

**Comment by qdm12** [comment]
As an example, I use the Gotify android app which connect to my Gotify container running on my server. DDNS-updater sends notifications to my Gotify container. You would also need a reverse proxy ideally to handle https.

For the email, it looks like you would need first of all a smtp server for ddns-updater to talk to, which would then send the email. Just configure it with `SHOUTRRR_ADDRESSES=smtp://username:password@host:port/?from=fromAddress&to=recipient1` for example.

**Reply by ghost** [reply]
> I use the Gotify android app which connect to my Gotify container running on my server. DDNS-updater sends notifications to my Gotify container

How did you configure **DDNS-Updater** to send send notifications via **Gotify**?

What I mean in particular is, is it possible to not use the **shoutrrr** service and instead interact directly with Gotify?

**Comment by rudedog7106** [comment]
> I've installed ddns-updater via TrueNAS Scale Apps, (App Version: 2.6.0, Chart Version: 1.0.27). I've setup two domains to update, however I can't seem to figure out how to setup/enable Shoutrrr to send notifications.
> 
> I'd like to send a simple notification of when an IP has been updated or failed to update. After looking at the [Shoutrrr services overview](https://containrrr.dev/shoutrrr/v0.8/services/overview/), for now email my preferred. But I'm not sure what to do on ddns-updater to get it done..
> 
> I've been looking at https://containrrr.dev/shoutrrr/v0.8/services/email/ however that's the only documentation/instructions I can find, so I'm basically unsure as I've never set something similar up before.

Did you ever setup DDNS to send email?

**Comment by tamimology** [comment]
I tried setting up notifications via Gotify. I have added the following env in my compose

`      - SHOUTRRR_ADDRESSES="gotify://sub.domain.com/GOTIFYTOKEN"`

Once I deploy, I get the following in the logs:
```
2026-08-05T07:08:10.616396390Z [ddns-updater] └── Shoutrrr
2026-08-05T07:08:10.616399106Z [ddns-updater]     ├── Default title: DDNS Updater
2026-08-05T07:08:10.616401808Z [ddns-updater]     └── Addresses
2026-08-05T07:08:10.616404561Z [ddns-updater]         └── gotify://sub.domain.com/GOTIFYTOKEN
2026-08-05T07:08:10.616627964Z [ddns-updater] 2026-08-05T17:08:10+10:00 INFO reading JSON config from file /updater/data/config.json
2026-08-05T07:08:10.617080790Z [ddns-updater] 2026-08-05T17:08:10+10:00 ERROR [shoutrrr] gotify: invalid gotify token "GOTIFYTOKEN"
```


What am I doing wrong here? I have tried this same token via curl and it worked just fine, which means the issue is with the container itself, or I am missing something in the way I am setting it up

Appreciate some help/guidance here
